docs/pages/enterprise/workflow/ssh-approval-jira-cloud.mdx

```diff
@@ -111,6 +111,15 @@ Run `./install` from `teleport-jira` or place the executable in `/usr/bin` or `/
 </TabItem>
 </Tabs>
 
+<Details title="Installing via Helm?">
```

Comment:
I think this would be clearest as a fourth `TabItem`. I think we should:

- Limit this `TabItem` to `helm repo add`
- Turn the instructions for starting the plugin into a `Tabs` component in the "Testing" section and add the `helm install` command there.
- Edit the "Configuration file" section to include the values file as another `TabItem`.

This way, users can have the same experience with this page for Helm as they would for other installation methods.

Reply:
I've changed the format a bit based on your comment, let me know if you'd change anything else there